tests/cupy_tests/lib_tests/test_shape_base.py

```diff
@@ -106,3 +106,11 @@ def test_apply_along_axis_invalid_axis():
         for axis in [-3, 2]:
             with pytest.raises(numpy.AxisError):
                 xp.apply_along_axis(xp.sum, axis, a)
+
+
+class TestApplyOverAxes(unittest.TestCase):
```

Comment:
Could you add another test for `axes.ndim == 0` for coverage?

Reply:
add `test_apply_over_axis_invalid_0darr`